src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -302,6 +302,14 @@
                 <source>An empty file is not allowed.</source>
                 <target>En tom fil är inte tillåten.</target>
             </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Domänen kunde inte hittas.</target>
```

Comment:
I feel like Sweden is often adopting technical terms from English, so "hosten" although perhaps not an official word would be the most recognized. Or I would say, domännamnet kunde inte hittas/uppslås.

Sent from my iPhone

> On 08 Sep 2015, at 00:01, Tobias Nyholm <notifications@github.com> wrote:
> 
> In src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf:
> 
> > @@ -302,6 +302,14 @@
> >                  <source>An empty file is not allowed.</source>
> >                  <target>En tom fil är inte tillåten.</target>
> >              </trans-unit>
> > +            <trans-unit id="79">
> > +                <source>The host could not be resolved.</source>
> > +                <target>Domänen kunde inte hittas.</target>
> The messge is shown when the dns check fails on a url-field. ref Do you think a normal user understands (or at least have a clue) what the error is if we are using "värddatorn"?
> 
> —
> Reply to this email directly or view it on GitHub.
>